script/test-carthage/CarthageTest/ContentView.swift

```diff
@@ -0,0 +1,18 @@
+// Created by Cal Stephens on 1/20/24.
+// Copyright © 2024 Airbnb Inc. All rights reserved.
+
+import Lottie
+import SwiftUI
+
+struct ContentView: View {
```

Comment:
Deleted the UIKit and AppKit CarthageTest apps in favor of a single cross-platform SwiftUI app that also supports visionOS and tvOS.